### DISCUSSION TITLE
How can I make the script running only for photo/media/files? and skipping key words?

### DISCUSSION THREAD

**Discussion by warmkenley** [aahnik/tgcf#6] [Q&A] [answered]
it there any information I can add in config so it can detect not to forward wordings message or to detect not sending some message with 'key words'?
Because one example, some groups have bot that will send welcome notice for every freshman, this is not looking so beauty for after forwarding all these repeat useless notice

**Comment by aahnik** [comment, marked as answer]
Hi @warmkenley, I completely understand your usecase.  And I will try to add those features in future.

------

By the way, I have another seperate script that **runs constantly ( live sync, real time forwarding )** . And can be deployed to Heroku, by a click of button. That is currently closed source for some reasons, and if you want that contact me on [Telegram @AahnikDaw](https://telegram.me/aahnikdaw)

**Comment by tissole** [comment]
Some filtering is really necessary. For example, excluding post that contains only chatting or copying only some types of files. In certain groups, 90% is just chatting and the files I want to copy represent 10% of all the posts. I have to copy tens of thousands of posts just to get those that I'm interested in. This feature will be really useful. Thank you for your work!

**Reply by aahnik** [reply]
This is a heavily requested feature. To forward only some kind of files like photos/gif/video/voice note/video note/pdf/etc.

Stay tight. Coming soon.

To get notified of updates join [tg_cf channel in telegram](https://telegram.me/tg_cf)

**Reply by ThatDevCoder** [reply]
Even I'm waiting for this feature. Can you give some walk-around of this feature till you officially release this feature. Coz this welcome message from bot while they join the group is really annoying as it also gets forwarded to me.
